src/clients/Device.js

```diff
@@ -146,9 +153,9 @@ class Device extends EventEmitter {
     await this.handshake();
   };
 
+  // TODO now we have handshake method and this._handshake module
+  // rename one of these.
   handshake = async (): Promise<*> => {
```

Comment:
now we have `handshake` method and `_handshake` module..so maybe rename module to HandshakeManager or something?